server/build.gradle

```diff
@@ -12,6 +12,7 @@ plugins {
     id "application"
     id "idea"
     id "groovy"
+    id "org.flywaydb.flyway" version "6.5.4"
```

Comment:
Any particular reason for going to the plugin? If this is handled by the application starting, as it is now, then we don't have to deal with configuration as a part of the build.

Reply:
Adding this plugin gives us some additional capabilities, such as being able to run `./gradlew flyway repair` to force a rebuild of the database when loading up a branch that has a different database state. It just adds some convenience.